resources/monitoring/values.yaml

```diff
@@ -943,9 +1111,8 @@ kubeProxy:
   service:
     port: 10249
     targetPort: 10249
-    selector:
```

Comment:
We should check this with GKE and Azure.

Reply:
kubeProxy.enabled is set to false, so I guess we are safe here